reference/forms/types/options/datetime_choice_translation_domain.rst.inc

```diff
@@ -0,0 +1,15 @@
+choice_translation_domain
```

Comment:
I would have. Unfortunately there is a conflict between added version : 2.7 for choice types but 2.8 for date types.
Do you know a way to override only the note in a block like this ?